doc/api/esm.md

```diff
@@ -734,6 +741,9 @@ prevent unintentional breaks in the chain.
 
 <!-- YAML
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/44710
+    description: Convert hook from asynchronous to synchronous.
```

Comment:
But does it actually have to be synchronous? Isn't the point of moving to another thread to keep loaders async?